README.md

```diff
@@ -102,6 +99,25 @@ interface Address {
   state?: "Texas" | "Alabama" | "California" | "other";
 }
 ```
+## Supported input
+These are the supported inputs.
+### AsyncAPI input
+The library expects the `asyncapi` property for the document to be sat in order to understand the input format.
+- Generate from a [parsed AsyncAPI document](https://github.com/asyncapi/parser-js)
```

Comment:
Is there any other option for parsing rather than JS? If so, would it be worth mentioning a few or just indicate the JS is an example of them?.

Reply:
@smoya `model-sdk` is written in JS and will be used in generator (also written in JS), so at the moment we cannot use parsed schema from another parser, I mean written in another language.

Reply:
Only solution to use package with parser written in another language is first parse schema, save parsed schema to file, and then read content from file and put in our `model-sdk`.

Reply:
> Only solution to use package with parser written in another language is first parse schema, save parsed schema to file, and then read content from file and put in our `model-sdk`.

Considering that https://github.com/asyncapi/parser-go is gonna become a thing at some point (with a thing I mean something used consciously), perhaps make sense to move to the model you suggest @magicmatatjahu.
However, the work of outputting to a file would be a task to be done by the parsers (in their code) rather than by this generator, right?

Reply:
@smoya Yes exactly. Custom parser should save parsed schema inside file, and then someone should read content from file and put it in model-sdk as I described. Using internally another parser without file-workaround is also possible, but then we must use web-assembly, and it will be very complex 😄

Reply:
I think I like the idea of files as input.

Reply:
But we must have in mind that if schema has circular references, then after saving it to file we will have missing data in some parts due to lack of circular refs.

Reply:
I too like the idea of file inputs @smoya would you mind creating a feature request for this? 🤔

Reply:
Done for this repository: https://github.com/asyncapi/generator-model-sdk/issues/97

Ideally a new issue should be created on each parser repository (JS and Go as per today). If you think it looks ok, I will proceed and create them.